tests/test_smoke.py

```diff
@@ -820,7 +820,7 @@ def test_new_bucket_creation_and_deletion(self, tmp_local_storage_obj,
 
     @pytest.mark.parametrize(
         'tmp_public_storage_obj, store_type',
-        [('s3://tcga-2-open', storage_lib.StoreType.S3),
```

Comment:
Can we keep this bucket in the test? For coverage.

Reply:
Great point - added!